package.json

```diff
@@ -14,13 +14,14 @@
   ],
   "repository": "git@github.com:evanshortiss/fh-health.git",
   "author": "Evan Shortiss",
+  "license": "Apache-2.0",
   "dependencies": {
     "async": "0.2.9",
     "fhlog": "^0.12.1"
```

Comment:
@evanshortiss when you get a chance to release the 0.13.0 - please update here too